contracts/TaskBook.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity ^0.4.4;
+
+import './AccountManager.sol';
+
+contract TaskBook is AccountManager {
+
+	uint private numTasks = 0;
+	uint constant maxSolvers = 10;
+	uint constant maxChallengers = 10;
+
+	event NewTask(uint taskID, uint minDeposit, uint blockNumber);
+	event SolveTask(uint indexed taskID, address solver, bytes32 taskData, uint minDeposit);
+	event SubmitSolution(uint taskID, bytes32 solution, uint minDeposit, bytes32 taskData);
```

Comment:
I prefer the "past tense" event names (`TaskSolved`, `SolutionSubmitted`), and the community seems to be converging on that standard. They've also started arguing for a `Log` prefix in front of the events (`LogTaskSolved`), but I'm not convinced it's necessary.

Reply:
Hmmm, glad you brought that up. Wasn't aware of that. Guess I could change:

NewTask => TaskCreated
SolveTask => SolverSelected
SubmitSolution => SolutionSubmitted